address-space-controller/src/main/java/io/enmasse/controller/AddressSpaceController.java

```diff
@@ -133,21 +133,21 @@ private void stop() {
     private void configureDefaultResources(NamespacedOpenShiftClient client, File resourcesDir) {
         String namespace = client.getNamespace();
         KubeResourceApplier.applyIfDifferent(new File(resourcesDir, "brokeredinfraconfigs"),
-                client.customResources(AdminCrd.brokeredinfraconfigs(), BrokeredInfraConfig.class, BrokeredInfraConfigList.class, DoneableBrokeredInfraConfig.class).inNamespace(namespace),
+                client.customResources(AdminCrd.brokeredInfraConfigs(), BrokeredInfraConfig.class, BrokeredInfraConfigList.class, DoneableBrokeredInfraConfig.class).inNamespace(namespace),
                 BrokeredInfraConfig.class,
                 Comparator.comparing(BrokeredInfraConfig::getVersion));
 
         KubeResourceApplier.applyIfDifferent(new File(resourcesDir, "standardinfraconfigs"),
-                client.customResources(AdminCrd.standardinfraconfigs(), StandardInfraConfig.class, StandardInfraConfigList.class, DoneableStandardInfraConfig.class).inNamespace(namespace),
+                client.customResources(AdminCrd.brokeredInfraConfigs(), StandardInfraConfig.class, StandardInfraConfigList.class, DoneableStandardInfraConfig.class).inNamespace(namespace),
                 StandardInfraConfig.class,
```

Comment:
why has this changed from standard to brokered?  was this intentional - it looks wrong

Reply:
It is wrong! And fixed.